modules/apps/frontend-js/frontend-js-clay-sample-web/src/main/java/com/liferay/frontend/js/clay/sample/web/internal/portlet/FrontendJSClaySampleWebPortlet.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.clay.sample.web.internal.portlet;
+
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet;
+
+import javax.portlet.Portlet;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Bryce Osterhaus
+ */
+@Component(
+	immediate = true,
+	property = {
+		"com.liferay.portlet.display-category=category.sample",
+		"com.liferay.portlet.header-portlet-css=/css/main.css",
```

Comment:
What's the reason for adding a header-portlet-css? I guess even for demo purposes we'd rather have `import from './foo.css'` in our JS, so we should be able to get rid of this and the `main.css` file.

Reply:
I left it in from the boilerplate I copy/pasted from. I'll remove all css for now since we likely won't use it